package.json

```diff
@@ -10,7 +10,8 @@
     "start": "next start",
     "lint:js": "next lint",
     "lint:css": "stylelint '**/*.scss'",
```

Comment:
What do you think about renaming this to `lint:scss` so that the command is more reflective of what it's doing?